cumulusci/tasks/github/publish.py

```diff
@@ -170,8 +199,8 @@ def _rename_files(self, zip_dir):
     def _create_commit(self, path):
         committer = CommitDir(self.target_repo, logger=self.logger)
         message = f"Published content from ref {self.ref}"
-        if "version" in self.options:
-            message += f'\n\nVersion {self.options["version"]}'
+        if "tag_name" in self.options:
+            message += f'\n\nTag {self.options["tag_name"]}'
```

Comment:
We should be specific here:

```suggestion
            message += f'\n\nTag {self.tag_name}'
```
Otherwise we could end up with `Tag latest_beta`.

Reply:
Actually, OK leave as is, but add a `FIXME`.

Reply:
I can make this change. It's better to use the instance reference to `tag_name` I feel.